muni_portal/core/signals.py

```diff
@@ -26,15 +27,26 @@ def queue_send_webpush_notification(notification_id):
                 vapid_claims={"sub": f"mailto:{settings.DEFAULT_FROM_EMAIL}"}
             )
             result.status_code = response.status_code
+            result.data = response.json() if response.json() else {}
```

Comment:
None is way better as "no value" than empty object, isn't it?

```suggestion
            result.data = response.json() if response.json() else None
```

Reply:
or actually just set it to response.json() - why did you use {}?

Reply:
To be sure that we updated value during request. If the field is None/Null we can't say for sure that we did this.